wiki/Beatmap_Editor/Test_mode/en.md

```diff
@@ -0,0 +1,31 @@
+---
+stub: true
+tags:
+  - editor
+  - beatmapping
+  - mapping
+---
+
+# Test mode
+
+The **test mode** is a feature of the [beatmap editor](/wiki/Beatmap_Editor). The mode is intended to allow creators to play their own [beatmaps](/wiki/Beatmaps) quickly and easily within the beatmap editor for testing purposes. The mode can be accessed via the `Test` button near the bottom-right corner of the editor.
+
+![Test mode button](img/test-mode-button.png "The test mode button in the beatmap editor")
+
+![Screenshot of test mode](img/test-mode-example.jpg "Screenshot of test mode using the autoplay function")
+
+*Note: playing a beatmap in test mode has been known to cause some lag or noticeable drops in framerate for some users.*
+
+Playing a beatmap in test mode differs from that of playing it normally in that it will not submit any scores, does not display any leaderboards, will not let users fail the beatmap, and will always display the following information on the left-hand side of the screen:
```

Comment:
```suggestion
Playing a beatmap in test mode differs from playing it normally, as it will not submit any scores, does not display any leaderboards, will not let users fail the beatmap, and will always display the following information on the left side of the screen:
```